src/Layouts/AuthLayout/index.js

```diff
@@ -0,0 +1,3 @@
+import AuthLayout from './AuthLayout';
+
+export default AuthLayout;
```

Comment:
export default in single line
ex- export { default as AuthLayout } from './AuthLayout'